lib/Backends/OpenCL/OpenCLDeviceManager.cpp

```diff
@@ -412,6 +416,225 @@ void OpenCLDeviceManager::returnRunCommandQueue(OpenCLCommandQueue &queue) {
   commandQueuePool_.returnCommandQueue(queue);
 }
 
+void OpenCLDeviceManager::copyInputsToDevice(
+    const RuntimeBundle &runtimeBundle, ExecutionContext *context,
+    runtime::OpenCLDeviceBindings *devBindings) {
+  TRACE_EVENT_SCOPE(context->getTraceContext(), TraceLevel::RUNTIME,
+                    "copyInputsToDevice");
+  bool profilingEnabled =
+      context->getTraceContext() &&
+      (context->getTraceContext()->getTraceLevel() & TraceLevel::COPY);
+  size_t sizeInBytes = runtimeBundle.getConstantWeightSize();
+  if (runtimeBundle.getConstants()) {
```

Comment:
I think we can get rid of this section. The constants should be coped to the device during addNetwork. I would guess this branch is never entered...

Reply:
Good catch this is totally redundant.